appdev/webapp.rst

```diff
@@ -0,0 +1,55 @@
+Creating a Webapp
+=================
+
+Webapps are small applications, that load content in a browser. This tutorial for ubuntu will just create a starter for a webpage, that can be displayed easily in a mobile device.
+
+
+alternate-webapp-generator
+--------------------------
+
+The alternate-webapp-generator will download all the stuff you need for a webapp and will add the the url, icon and a lot of meta-information to the app.
+In the end a .click package will be created, which can be uploaded to the OpenStore
+
+Download the following zip-file and unzip
+::
+
+    https://github.com/Winael/ubuntu/blob/master/ubuntuphone/webapps/Reboot/alternate-webapp-generator.zip
+
+Make the app-creator executable:
+::
+
+    chmod +x /path/to/alternate-webapp-generator.sh
+
+Now we can start and edit the config.cfg.
+::
+
+    # config.cfg
+
+    export namespace="namespace"
+    
```

Comment:
Do we want the whitespace here?

Reply:
Its from the template file, but i can remove it from here - i dont mind

Reply:
I meant the line breaks, actually, not the space. A space between the comment operator and the comment is generally considered a good practice, so i guess that should stay